### PR TITLE
Use origami-ci

### PR DESCRIPTION
**Pull request by chee**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"private": true,
```
Comment (JakeChampion) [inline package.json]: What does `occ` do when it sees `private:"true"`?
Does it remove it?

Comment (chee) [reply package.json]: `occ` generates a new `package.json` from the `bower.json`, and only merges specific fields from an existing `package.json`. it doesn't even look at `private`. reference: https://github.com/Financial-Times/origami-component-converter/blob/fd7a8813a6d3437f4a8f54c5944f32c88fb51f14/lib/npm.js#L59-L78

**Comment by chee** [comment]
i just noticed the spec says it "must not" include a version property: https://origami.ft.com/spec/v1/components/#npm

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,27 +2,18 @@ version: 2
...
-      - run: npm i -g origami-build-tools@^7
-      - run: $HOME/.local/bin/obt install
-      - run: $HOME/.local/bin/obt demo --demo-filter pa11y --suppress-errors
-      - run: $HOME/.local/bin/obt verify
-      - run: $HOME/.local/bin/obt test
-      - run: git clean -fxd
-      - run: npx occ 0.0.0
-      - run: $HOME/.local/bin/obt install --ignore-bower
-      - run: $HOME/.local/bin/obt test --ignore-bower
+      - run: npm install --only=dev
```
Comment (chee) [inline .circleci/config.yml]: @JakeChampion what do you think of using `--only=dev` here ?

Comment (JakeChampion) [reply .circleci/config.yml]: Very good idea, it'll help find components which may accidentally bring in prod dependencies via npm

**Comment by origamiserviceuser** [comment]
o-colors bundle size difference from 4.10.2 to 238db68351d9a4e4ce77fadec16355c69021f516
No bundle size difference found.

**Comment by chee** [comment]
@notlee look at that!!

**Pull request merged**